src/components/Loading/Loading.css

```diff
@@ -0,0 +1,129 @@
+/**
+ * ----------------------------------------
+ * Début règle animation heartbeat
+ * ----------------------------------------
+ */
+  @keyframes heartbeat {
+    from {
+      -webkit-transform: scale(1);
+              transform: scale(1);
+      -webkit-transform-origin: center center;
+              transform-origin: center center;
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+    10% {
+      -webkit-transform: scale(0.91);
+              transform: scale(0.91);
+      -webkit-animation-timing-function: ease-in;
+              animation-timing-function: ease-in;
+    }
+    17% {
+      -webkit-transform: scale(0.98);
+              transform: scale(0.98);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+    33% {
+      -webkit-transform: scale(0.87);
+              transform: scale(0.87);
+      -webkit-animation-timing-function: ease-in;
+              animation-timing-function: ease-in;
+    }
+    45% {
+      -webkit-transform: scale(1);
+              transform: scale(1);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+  }
+  /**
+   * ----------------------------------------
+   * Fin règle animation heartbeat
+   * ----------------------------------------
+   */
+  
+  
+  /**
+   * ----------------------------------------
+   * Début règle animation loading
+   * ----------------------------------------
+   */
+   @keyframes bouncing-loader {
+    to {
+      opacity: 0.1;
+      transform: translate3d(0, -1rem, 0);
+    }
+  }
+  /**
+   * ----------------------------------------
+   * Fin règle animation loading
+   * ----------------------------------------
+   */
+   
+html{ height:90%; width: 100%; }
```

Comment:
Don't use global styles with html selectors in a component's css file. It breaks a component encapsulation and might break other components or pages.